integration-test/facebook-sdk-schema.spec.js

```diff
@@ -8,7 +8,9 @@ const testSite = 'https://privacy-test-pages.glitch.me/privacy-protections/click
 test.describe('Facebook SDK schema', () => {
     test('CTL: Facebook SDK schema hasn\'t changed', async ({ page }) => {
         await routeFromLocalhost(page)
-        await page.goto(testSite, { waitUntil: 'networkidle' })
```

Comment:
Waiting for 'networkidle' causes the YouTube test to timeout, so I wait for the API to be ready instead. That worked nicely, so I figured we should do that here too. It speeds the test up quite a bit.